ci/tests/general/general_vdisk.py

```diff
@@ -319,3 +321,26 @@ def is_volume_synced_up_to_snapshot(vdisk, snapshot_id):
             'is_volume_synced_up_to_snapshot failed for vdisk: {0}'.format(vdisk.name)
 
         return result
+
+    @staticmethod
+    def snapshot_disk(diskguid, metadata):
+        """
+        Create snapshot for vdisk
+        :param diskguid: guid of the vdisk to snapshot
+        :param metadata: metadata = {'label': 'snp1',
+                                    'is_consistent': True,
+                                    'timestamp': time.time(),
+                                    'machineguid': vm disk is attached to,
+                                    'is_automatic': False,
+                                    'is_sticky': False}
+        :return: None
+        """
+        VDiskController.create_snapshot(diskguid, metadata)
+
+    @staticmethod
+    def start_scrub_work():
+        """
```

Comment:
as this is a one-liner I suggest just putting directly in the code, without creating a specific method